test/integration/01-setup/01-connection.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { setupMongoConnection } from "src/mongodb"
```

Comment:
what is the reason to have the filename being `.spec.ts`?

Reply:
It is a common naming convention and It is easier to filter from helpers and configuration files. We discussed it in the 1st stage PR https://github.com/GaloyMoney/galoy/pull/310#pullrequestreview-701722944